plugin/pkg/scheduler/algorithm/priorities/priorities.go

```diff
@@ -89,7 +89,42 @@ func LeastRequestedPriority(pod *api.Pod, podLister algorithm.PodLister, minionL
 
 	list := algorithm.HostPriorityList{}
 	for _, node := range nodes.Items {
-		list = append(list, calculateOccupancy(pod, node, podsToMachines[node.Name]))
+		list = append(list, calculateResourceOccupancy(pod, node, podsToMachines[node.Name]))
+	}
+	return list, nil
+}
+
+func min(l, r int64) (m int64) {
+	m = r
+	if l < r {
+		m = l
+	}
+	return m
+}
+
+// See comment for DumbSpreadingPriority()
+const dumbSpreadingDenominator int64 = 10
+
+// DumbSpreadingPriority is a priority function that favors nodes with fewer pods.
+// It works like LeastRequestedPeriority but instead of using 10 * percentage of machine free by resource,
+// it uses 10 * percentage of machine free by pod, with "percentage of machine free by pod" claculated as
+// (dumbSpreadingDenominator - number of pods already on the node + 1) / dumbSpreadingDenominator.
+// dumbSpreadingDenominator serves like the machine capacity in LeasRequestedPriority but is chosen
+// so that we equate one pod with a reasonable amount of resources when we combine all the scores together.
+func DumbSpreadingPriority(pod *api.Pod, podLister algorithm.PodLister, minionLister algorithm.MinionLister) (algorithm.HostPriorityList, error) {
+	nodes, err := minionLister.List()
+	if err != nil {
+		return algorithm.HostPriorityList{}, err
+	}
+	podsToMachines, err := predicates.MapPodsToMachines(podLister)
+
+	list := algorithm.HostPriorityList{}
+	for _, node := range nodes.Items {
+		npods := int64(len(podsToMachines[node.Name]))
+		list = append(list, algorithm.HostPriority{
+			Host:  node.Name,
+			Score: calculateScore(min(npods+1, dumbSpreadingDenominator), dumbSpreadingDenominator, node.Name),
```

Comment:
Do we want to have dumbSpreadingDenominator equal to 10? This basically means that if on node-1 we have 11 pods and on node-2 we have 30 pods, those two will have the same scoring function. Do we want that? Don't we want to have it equal to 30, which is our target for "pods per node" in v1.0?